src/components/CheckboxInput/CheckableInput/CheckableInput.tsx

```diff
@@ -14,7 +14,7 @@ export type Props = PropsWithSpread<
     /**
      * The label for the input element.
      */
-    label: ReactNode;
+    label?: ReactNode;
```

Comment:
I think the label is required and shouldn't be optional.

Reply:
yeah true, since we are already doing a falsey check for label it can remain as a required prop